app/views/deploys/_buddy_check.html.erb

```diff
@@ -7,7 +7,8 @@
     <div class="deployer-stop">
       <%= stop_button %>
       or
-      <%= link_to "Bypass", buddy_check_project_deploy_path(@project, @deploy), method: :post, class: "btn btn-danger" %>
+      <% confirm = "Are you sure this is an emergency and you cannot find a buddy?\n#{details}" %>
```

Comment:
since details is marked as html_safe above would probably make sense to [strip_tags](http://api.rubyonrails.org/classes/ActionView/Helpers/SanitizeHelper.html#method-i-strip_tags) here